regexes/device/mobiles.yml

```diff
@@ -13050,9 +13112,14 @@ Vernee:
 
 # Vertu
 Vertu:
-  regex: 'Vertu[ ]?([a-z0-9]+)'
+  regex: 'Vertu[ ]?([a-z0-9]+)|ASTER P[/;) ]'
   device: 'feature phone'
-  model: '$1'
+  models:
+    - regex: 'ASTER P[/;) ]'
+      device: 'smartphone'
+      model: 'Aster P'
+    - regex: 'Vernee[ _-]([a-z0-9_-]+)'
```

Comment:
Shouldn't that be `Vertu`? Seems there actually was no test for a Vertu device. Maybe we could add one with e.g. this user agent: `Vertu Signature/2.0 (08w31) IMEI/XXXXXXXXXXXXXXX Profile/MIDP-2.0 Configuration/CLDC-1.1`

Reply:
You can't find out the model without the first IMEI characters

Reply:
Then we can use `Vertu Signature/2.0 (502.06) IMEI/355711020080019 Profile/MIDP-2.1 Configuration/CLDC-1.1`
Nevertheless I think `Vernee` is a typo in the regex, isn't it?

Reply:
Yes that's right `Vernee` is a typo